sabnzbd/utils/checkdir.py

```diff
@@ -103,6 +103,30 @@ def isFAT(check_dir):
     return FAT
 
 
+def directory_is_writable(mydir, myfilename):
+    filename = os.path.join(mydir, myfilename)
+    if os.path.exists(filename):
+        try:
+            os.remove(filename)
+        except:
+            return False
+    try:
+        with open(filename, "w") as f:
+            f.write("Some random content")
+        os.remove(filename)
+        return True
+    except:
+        return False
+
+
+def directory_is_writable_basic(mydir):
+    return directory_is_writable(mydir, "sab_test.txt")
```

Comment:
Do we really need a basic check? Don't we just want to check the expected "regular" case?

Reply:
Ok!